pkg/controller/controller.go

```diff
@@ -288,16 +291,29 @@ func (c *Controller) monitor(watchVersion string) (<-chan *Event, <-chan error)
 				}
 
 				if st != nil {
-					if st.Code == http.StatusGone { // event history is outdated
-						errCh <- ErrVersionOutdated // go to recovery path
+					resp.Body.Close()
+
```

Comment:
Can you add a logging here StatusGone?
It's not normal path. If LIST-ing/rebuilding cache is expensive, users might need to know what happen?